### PR TITLE
Add percent change support

### PR DESCRIPTION
**Pull request by francoiscampbell**
**What kind of change does this PR introduce?**
New feature

**Did you add tests for your changes?**
Updated snapshot shows percent change

**If relevant, link to documentation update:**
Will update docs if this PR is accepted

**Summary**
First steps towards addressing #8

**Does this PR introduce a breaking change?**
No, only the output string is modified

### CODE REVIEW

File: src/app/analyze/analyze.test.mockdata.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+export const mockFileResults = {
+    oneFileLarger: [
+        {
+            size: 125,
+            baseBranchSize: 100,
+        },
+    ],
+    oneFileSmaller: [
+        {
+            size: 200,
```
Comment (jakebolam) [inline src/app/analyze/analyze.test.mockdata.js]: This file is larger?

Comment (francoiscampbell) [reply src/app/analyze/analyze.test.mockdata.js]: The mock new size is bigger than the mock base branch size. The actual file size doesn't matter, just whether it grew or shrunk compared to the base branch

Comment (jakebolam) [reply src/app/analyze/analyze.test.mockdata.js]: This test case is the same as `oneFileLarger` no? size > baseBranchSize.
Should size be smaller than baseBranchSize here?

Comment (francoiscampbell) [reply src/app/analyze/analyze.test.mockdata.js]: :man_facepalming: yeah. The test is wrong too so i didn't notice

**Review by jakebolam — APPROVED** [review]

**Pull request merged**